hmac.h

```diff
@@ -36,7 +36,19 @@ typedef sha256_context_t dtls_sha256_ctx;
 #define dtls_sha256_update(Ctx,Input,Len) sha256_update((Ctx), (Input), (Len))
 #define dtls_sha256_final(Buf,Ctx)        sha256_final((Ctx), (Buf))
 
-#else /* RIOT_VERSION */
+#elif defined ESPIDF_VERSION
+#include "sodium/crypto_hash_sha256.h"
+
+typedef crypto_hash_sha256_state dtls_hash_ctx;
+typedef crypto_hash_sha256_state dtls_sha256_ctx;
+#define DTLS_HASH_CTX_SIZE sizeof(crypto_hash_sha256_state)
+#define DTLS_SHA256_DIGEST_LENGTH (32)
```

Comment:
I suggest using `crypto_hash_sha256_BYTES` here instead of the magic number.

Reply:
Fixed.